bin/start_backend.sh

```diff
@@ -141,6 +141,10 @@ export LIBHDFS_OPTS=$final_java_opt
 # Otherwise, JVM will overwrite the signal handlers for SIGINT and SIGTERM.
 export LIBHDFS_OPTS="$LIBHDFS_OPTS -Xrs"
 
+# add custom_libs to CLASSPATH
+for f in $STARROCKS_HOME/custom_lib/*.jar; do
+    CLASSPATH=$f:${CLASSPATH}
+done
```

Comment:
why we need this?

Reply:
When using a higher version of Oracle dependency, a character set exception will be reported due to the lack of orai18n. jar dependency as follows:
"Non supported character set (add orai18n.jar in your classpath): ZHS16GBK"
In order to make upgrading operations more user-friendly for users, we have chosen to add this directory.

Reply:
> why we need this?

When using a higher version of Oracle dependency, a character set exception will be reported due to the lack of orai18n. jar dependency as follows:
"Non supported character set (add orai18n.jar in your classpath): ZHS16GBK"
In order to make upgrading operations more user-friendly for users, we have chosen to add this directory.